### PR TITLE
Added one.alynx.FlipClock.

### PR DESCRIPTION
**Pull request by AlynxZhou**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: one.alynx.FlipClock.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      - "--disable-static"
+      # The SDK contains FreeType and HarfBuzz, so prevent from building them.
+      - "--disable-freetype-builtin"
+      - "--disable-harfbuzz-builtin"
+    cleanup:
+      # Those files are for developing, not for running.
+      - "/include"
+      - "/lib/pkgconfig"
+      - "/lib/*.la"
+      - "/lib/*.a"
```
Comment (hfiguiere) [inline one.alynx.FlipClock.yaml:23]:
```suggestion
      - "*.la"
      - "*.a"
```

File: one.alynx.FlipClock.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+app-id: one.alynx.FlipClock
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: flipclock
+modules:
+  # The SDL_ttf in `shared-modules` repo is for SDL1,
+  # and the SDK only contains SDL2 but has no SDL2_ttf,
+  # so I will build and bundle SDL2_ttf by myself.
+  - name: SDL2_ttf
```
Comment (hfiguiere) [inline one.alynx.FlipClock.yaml:10]: 2.0.15 is in the SDK. Isn't it enough?

Comment (AlynxZhou) [reply one.alynx.FlipClock.yaml:10]: Sorry, don't know why I never see it yesterday.

File: one.alynx.FlipClock.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        # Yes, SDL2_ttf is actually in the `SDL_ttf` repo.
+        url: https://github.com/libsdl-org/SDL_ttf/archive/release-2.0.18.tar.gz
+        sha512: 4baa0917e5a3b7daf046eb624369202dbc4be5db6197807bb0228252e290fd62d9daa2063f177296479e654035804b72afd992a556b4299a01657710c8d7ec2a
+  - name: flipclock
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://github.com/AlynxZhou/flipclock/archive/v2.8.7.tar.gz
+        sha512: b9f91a1a32ce4dc54a5a3197328c6486c5b54e4fcda61aa5d1578749bcd4bfb9cc80dc9fb146dda8b631750d4740384e3d26ee44f83ac5c3a53deaf33f0b6dbf
+finish-args:
```
[the remark this answers is not in the document]
Comment (AlynxZhou) [reply one.alynx.FlipClock.yaml:6]: Done

File: one.alynx.FlipClock.yaml (hunk 2 above)
Comment (AlynxZhou) [reply one.alynx.FlipClock.yaml]: Please check my newest commit.

**Comment by hfiguiere** [comment]
you can also:

bot, build one.alynx.FlipClock

**Pull request closed without merging**